source/guides/core-concepts/interacting-with-elements.md

```diff
@@ -129,12 +129,14 @@ Before interacting with an element, we will *always* scroll it into view (includ
 This scrolling logic only applies to {% urlHash "commands that are actionable above" Actionability %}. **We do not scroll elements** into view when using DOM commands such as {% url "`cy.get()`" get %} or {% url "`.find()`" find %}.
 {% endnote %}
 
-The scrolling algorithm works by scrolling the top, leftmost point of the element we issued the command on to the top, leftmost scrollable point of its scrollable container.
+By default, the scrolling algorithm works by scrolling the top, leftmost point of the element we issued the command on to the top, leftmost scrollable point of its scrollable container.
 
 After scrolling the element, if we determine that it is still being covered up, we will continue to scroll and "nudge" the page until it becomes visible. This most frequently happens when you have `position: fixed` or `position: sticky` navigation elements which are fixed to the top of the page.
 
 Our algorithm *should* always be able to scroll until the element is not covered.
 
+To change the position in the viewport to where we scroll an element, you can use the {% url `scrollBehavior` configuration#Actionability %} configuration option. This can be useful if the element is covered up when aligned to the top of the viewport, or if you just prefer seeing the element centered during commands. Accepted values are `'center'`, `'top'`, `'bottom'`, `'nearest'`, and `false`, with `false` disabling scrolling altogether.
```

Comment:
Don't want to confuse the scrolling that happens when you see the commands run versus the scroll that happens during actionable commands.

```suggestion
To change the position in the viewport to where we scroll an element, you can use the {% url `scrollBehavior` configuration#Actionability %} configuration option. This can be useful if the element is covered up when aligned to the top of the viewport, or if you just prefer the element to be centered during scrolling of action commands. Accepted values are `'center'`, `'top'`, `'bottom'`, `'nearest'`, and `false`, with `false` disabling scrolling altogether.
```